scripts/start.sh

```
@@ -0,0 +1,46 @@
+
+# 3) Start exactly ONE server, log to file
+rm -f server.pid server.log
+(
+  node dist/index.js \
+  || npx --yes tsx server/index.ts
+) > server.log 2>&1 &
+echo $! > server.pid
+
+# 4) Readiness check
+ok=0
+for i in {1..60}; do
```

Shellcheck (0.10.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)